pkgs/by-name/la/labwc-menu-generator/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, glib
+, gitUpdater
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "labwc-menu-generator";
+  version = "unstable-2023-10-31";
+
+  src = fetchFromGitHub {
+    owner = "labwc";
+    repo = finalAttrs.pname;
+    rev = "d7c81071f8b121ef83da32ae3fa16155d1a2ced9";
+    hash = "sha256-gZ0TuSVJwcKW4orawSmRQvoCfrpb8yLXlv81qCR86MU=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    glib
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp -a labwc-menu-generator $out/bin
+    runHook postInstall
+  '';
+
+  passthru.updateScript = gitUpdater { };
+
+  meta = {
+    homepage = "https://github.com/labwc/labwc-menu-generator";
+    description = "Menu generator for labwc";
+    mainProgram = "labwc-menu-generator";
+    license = lib.licenses.gpl2Only;
+    platforms = lib.platforms.unix;
+    maintainers = with lib.maintainers; [ romildo ];
```

Comment:
```suggestion
    license = lib.licenses.gpl2Only;
    mainProgram = "labwc-menu-generator";
    maintainers = with lib.maintainers; [ AndersonTorres romildo ];
    platforms = lib.platforms.unix;
```

Reply:
Added AndersonTorres to the list of maintainers.